.changeset/silly-coats-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+"@stargatefinance/stg-evm-v2": major
```

Comment:
No changeset is neccessary, and specifically the 'major' flag is improperly used